long-reach_0.0.7/data-final-fixes.lua

```diff
@@ -1,2 +1,5 @@
-data.raw.player.player.build_distance = 10000
-data.raw.player.player.reach_distance = 10000
+-- Rational: Pulling items out of machines is a nice timesaver, building 
+-- lets you do things like spam turrets into biter nests, so I think 
+-- building should be shorter range than adding / removing items
+data.raw.player.player.build_distance = 150
+data.raw.player.player.reach_distance = 300
```

Comment:
150 units is already a perfectly safe distance for spamming turrets in biter nests. For simplicity, please remove the rationale and set these to the same value. So long as it covers the entire viewport, I don't really care what the amount is.